build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rm -rf CMakeFiles
+rm CMakeCache.txt
+cmake .
+make 
```

Comment:
scripts should all go in the `scripts` dir.

Reply:
reconsidering, the scripts should probably go in `scripts/osx`

Reply:
sounds good.